tsconfig.json

```diff
@@ -8,7 +8,7 @@
     "declaration": false,
     "moduleResolution": "node",
     "experimentalDecorators": true,
-    "target": "es5",
+    "target": "es2015",
```

Comment:
nitpick: Was it deliberate to change compodoc tsconfig target in this PR to es2015?

Reply:
I might have rushed things. I think this still should be done, but maybe in a separate commit on the main branch, right?